src/main/java/io/anserini/search/queryencoder/SpladePpEdMsmarcoPassageEncoder.java

```diff
@@ -0,0 +1,161 @@
+package io.anserini.search.queryencoder;
+
+import ai.djl.modality.nlp.DefaultVocabulary;
+import ai.djl.modality.nlp.bert.BertFullTokenizer;
+import ai.onnxruntime.OnnxTensor;
+import ai.onnxruntime.OrtEnvironment;
+import ai.onnxruntime.OrtException;
+import ai.onnxruntime.OrtSession;
+import org.apache.commons.io.FileUtils;
+
+import java.io.*;
+import java.net.URL;
+import java.nio.file.Path;
+import java.util.*;
+
+public class SpladePpEdMsmarcoPassageEncoder extends QueryEncoder{
+    static private final String MODEL_URL = "https://dl.dropboxusercontent.com/s/8eq7pjkm1lrwe1r/spladePpEd.onnx?dl=0";
+
+    static private final String VOCAB_URL = "https://dl.dropboxusercontent.com/s/2kgkvw6gm37ghc8/vocab.txt?dl=0RYd";
+
+    static private final String MODEL_NAME = "spladePpEdEncoder.onnx";
+
+    static private final String VOCAB_NAME = "vocab.txt";
```

Comment:
Since all three encoders use the same vocab list, should we name them using the same name? (i.e., so the user would only need to download one file instead of 3)

Reply:
Like naming all three vocab as 'WordpieceVocab.txt'

Reply:
Space is free... so let's make three copies and give them different names?

Reply:
Sounds good